packages/matchbox/src/components/ListBox/useOptionConstructor.js

```diff
@@ -77,15 +77,15 @@ function useOptionConstructor({ options, value, onSelect, open, placeholder }) {
   }, [focused]);
 
   React.useLayoutEffect(() => {
-    if (keysSoFar) {
+    if (startsWith) {
       const index = options.findIndex(option =>
-        option.props.value.toLowerCase().includes(keysSoFar),
+        option.props.value.toLowerCase().includes(startsWith),
```

Comment:
Oh i meant `.startsWith` instead of `.includes` not `keysSoFar`

Reply:
> Oh i meant `.startsWith` instead of `.includes` not `keysSoFar`

Oh yea that makes more sense

Reply:
Oh yea that makes more sense